KLSadd_insertion/README.md

```diff
@@ -15,5 +15,110 @@ NOTE: when you run updateChapters.py or linetest.py you need the KLSadd.tex file
 
 **THIS MEANS YOU SHOULD NOT RUN updateChapters.py OR linetest.py FROM YOUR GIT DIRECTORY. THE CHAPTER FILES ARE NOT PUBLIC DOMAIN, DO NOT COMMIT THEM! COPY THE FILES OVER INTO A SEPERATE DIRECTORY AND THEN COPY THEM BACK OVER WHEN YOU ARE READY TO COMMIT** 
```

Comment:
Correct this information - simply put the chapter files into the .gitignore instead of moving them back and forth